imagededup/handlers/search/brute_force_cython_ext.pyx

```diff
@@ -0,0 +1,44 @@
+#cython: boundscheck=False, wraparound=False
+import cython
+from libcpp.vector cimport vector
+from libcpp.string cimport string
+
+
+ctypedef unsigned long long ull
+
+cdef extern int __builtin_popcountll(unsigned long long) nogil
+
+cdef vector[string] all_filenames
+cdef vector[ull] all_phashes
+
+
+def add(ull phash, string filename):
+    all_phashes.push_back(phash)
+    all_filenames.push_back(filename)
+
+
+def query(ull query_phash, unsigned int maxdist):
+    matches = []
+    cdef unsigned int dist
+    cdef string filename
+    cdef ull phash
+
+    cdef ull i
+
+    for i in range(all_phashes.size()):
+        phash = all_phashes[i]
+        filename = all_filenames[i]
```

Comment:
possible micro-optimisation here, we don't need the filename yet

Reply:
Agreed, let's move filename variable assignment inside the if condition that follows this LOC.